src/modules/app/StateType.js

```diff
@@ -37,6 +39,11 @@ export type FileCacheStateType = {
   |}
 }
 
+export type ErrorStateType = {|
+  +error: true,
```

Comment:
If this is a boolean then this should be called something like `loadingFailed` `errorHappened`

Reply:
I just used the flow terminology

Reply:
Which flow terminology?

Reply:
The one from  the flow docs, https://flow.org/en/docs/types/unions/#toc-disjoint-unions-with-exact-types :D